Tests/Unit/NetworkServer/BasicsStation/DownstreamSenderTests.cs

```diff
@@ -77,33 +83,41 @@ public async Task SendDownstreamAsync_Succeeds_WithValidDownlinkMessage_ClassADe
             Assert.Contains(@"""RX1DR"":5,", actualMessage, StringComparison.InvariantCulture);
             Assert.Contains(@"""RX1Freq"":868500000,", actualMessage, StringComparison.InvariantCulture);
             Assert.Contains(@"""RX2DR"":0,", actualMessage, StringComparison.InvariantCulture);
-            Assert.Contains(@"""RX2Freq"":869525000,", actualMessage, StringComparison.InvariantCulture);
+            Assert.Contains(@"""RX2Freq"":869500000,", actualMessage, StringComparison.InvariantCulture);
             Assert.Contains(@"""xtime"":123456,", actualMessage, StringComparison.InvariantCulture);
-            if (rfchHasValue)
-            {
-                Assert.Contains(@"""rctx"":1", actualMessage, StringComparison.InvariantCulture);
-            }
-            else
-            {
-                Assert.DoesNotContain("rctx", actualMessage, StringComparison.InvariantCulture);
-            }
+           
             Assert.Contains(@"""priority"":0", actualMessage, StringComparison.InvariantCulture);
+
+            // THIS NEED TO BE REWORKED
+            // rctx need to be reworked
```

Comment:
The current processing we are doing with rctx is wrong we need to rework this as part of  #1125